ote/src/cljs/ote/ui/main_header.cljs

```diff
@@ -262,7 +262,7 @@
                     style-topnav/topnav-dropdown-link)
                   {:target "_blank"})]]
          [:li
-          [linkify "http://bit.ly/nap-palaute" (tr [:common-texts :navigation-give-feedback])
+          [linkify "mailto:joukkoliikenne@traficom.fi?subject=Palautetta%20NAP-palvelusta%20" (tr [:common-texts :navigation-give-feedback])
```

Comment:
Would this work if you move that address to translation files?

Reply:
Moved the links to translation files.